src/renderer/features/assets/AssetsPortfolio/lib/tokensService.ts

```diff
@@ -0,0 +1,81 @@
+import tokensProd from '@shared/config/tokens/tokens.json';
+import tokensDev from '@shared/config/tokens/tokens_dev.json';
+import { sumValues } from '@shared/api/network/service/chainsService';
+import type { Account, AccountId, Balance, ChainId, TokenAsset, TokenBalance } from '@shared/core';
+import { ZERO_BALANCE, totalAmount } from '@shared/lib/utils';
+import { balanceUtils } from '@entities/balance';
+import { accountUtils } from '@entities/wallet';
+import { AssetChain } from './types';
+
+const TOKENS: Record<string, any> = {
```

Comment:
We use enum-like naming for such constants
```javascript
const Tokens = { DEV: ..., PROD: ... };
```

Reply:
In this case, I suggest changing construction in `chainsService` as well otherwise we'll have a different approach for the same thing
```
const CHAINS: Record<string, any> = {
  chains: chainsProd,
  'chains-dev': chainsDev,
};
```
Sounds good?

Reply:
Yeah, agree

Reply:
Update: we use it this way to avoid type issues with `process.env.TOKENS_FILE`